### DISCUSSION TITLE
What are the next steps?

### DISCUSSION THREAD

**Discussion by lutzhamel** [lutzhamel/asteroid#55] [General]
Development:
* Features
* Bugfixes
* Releases
* Libraries
* Example Programs
   * Pattern matching as a programming paradigm
   * First-class patterns high-lights

Documentation:
* User Guide
* Language Reference
* Library Reference
* Anything else?

Community:
* Outreach
* Community building
* Workshops?
* Slack Channel

**Comment by SupurCalvinHiggins** [comment]
What do you think about adding a basic TAP module and rewriting the tests?

**Comment by lutzhamel** [comment]
@SupurCalvinHiggins We have a slack channel to discuss Asteroid issue.  It works much better because all the discussions are centralized.  Please send me your email so I can invite you to the slack channel.

FYI:  the Python platform is temporary.  We are planning to move the whole thing to C or rather, the Asteroid compiler will be self-hosting and the runtime will be written in C.  So I am a little cautious about leaning on Python too heavily for the implementation etc.  Our current implementation uses Python in a very vanilla way...

**Comment by lutzhamel** [comment]
BTW, my email is lutzhamel@uri.edu

**Reply by SupurCalvinHiggins** [reply]
@lutzhamel Thank you for the reply! My email is calvin_higgins2@uri.edu.
